### ISSUE TITLE
How can I configure an OTP server ?

### ISSUE THREAD

**Issue by fabrice1236** [HSLdevcom/digitransit-ui#4752]
I'm new to digitransit and OTP, and I'm trying to integrate my own OTP server into digitransit. I tried to put my server URL into API_URL, but it tried to get finland data inside of it, which I don't have, is there a way to fix that ? And are there other settings I should also change ?

**Comment by vesameskanen** [comment]
Environment variable OTP_URL defines the address of the OpenTripPlanner service. For example:

OTP_URL=http://localhost:9080/otp/routers/fabricesowndata/

**Comment by fabrice1236** [comment]
Do I also need to change the MAP_URL and GEOCODING_BASE_URL variables ?

**Comment by optionsome** [comment]
Yes but depending on what type of a map and geocoding service you want to use, you might also need to edit the code to support those map and geocoding providers. We don't provide map or geocoding APIs that would cover areas outside of Finland unfortunately, so you need to find some alternatives or host your own.
